.github/workflows/windows_2019.yml

```diff
@@ -0,0 +1,75 @@
+name: Windows-2019
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 5 * * *'
+
+env:
+  VCPKG_PKGS: >-
+    boost-graph eigen3 console-bridge
+
+jobs:
+  windows_ci:
+    name: Windows-2019
+    runs-on: windows-2019
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@master
+      with:
+        path: workspace/src/descartes_light
+
+    - name: checkout-vcpkg
+      uses: actions/checkout@master
+      with:
+        path: vcpkg
+        repository: microsoft/vcpkg
+
+    - name: bootstrap-vcpkg
+      working-directory: vcpkg
+      run: bootstrap-vcpkg.bat
+
+    - name: vcpkg-dry-run
+      working-directory: vcpkg
+      shell: cmd
+      run: |
+        vcpkg install --dry-run --triplet x64-windows ${{ env.VCPKG_PKGS }} > vcpkg_dry_run.txt
+
+    - name: cache-vcpkg-archives
+      if: startsWith(github.ref, 'refs/tags/v') != true
+      id: cache-vcpkg-archives
+      uses: pat-s/always-upload-cache@v3
+      with:
+        path: C:\Users\runneradmin\AppData\Local\vcpkg\archives
+        key: ${{ runner.os }}-x64-vcpkg-${{ hashFiles('vcpkg/vcpkg_dry_run.txt') }}-main
+
+    - name: install-depends
+      shell: cmd
+      run: |
+        vcpkg integrate install
+        python -m pip install vcstool -q
+        python -m pip install colcon-common-extensions -q
+        vcpkg install --triplet x64-windows ${{ env.VCPKG_PKGS }}
+
+    - name: configure-msvc
+      uses: ilammy/msvc-dev-cmd@master
+      with:
+        arch: x64
+
+    - name: build-workspace
+      working-directory: workspace
+      shell: cmd
+      run: |
+        vcs import src < src/descartes_light/dependencies.rosinstall
+        colcon build --event-handlers console_direct+ --cmake-args -DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Debug -DDESCARTES_ENABLE_TESTING=ON
+        if %ERRORLEVEL% GEQ 1 exit 1
+
+    - name: run-test
+      working-directory: workspace
+      shell: cmd
+      run: |
+        colcon test --event-handlers console_direct+ --return-code-on-test-failure
```

Comment:
same here `if %ERRORLEVEL% GEQ 1 exit 1` I am also adding after the next command, but not sure if it is really needed.